tincr/io/design/tincr_checkpoints.tcl

```diff
@@ -509,44 +562,100 @@ proc write_net_routing { net_list channel } {
         puts $channel "INTERSITE VCC [join $vcc_sinks]"
     }
     
-    if {$vcc_route_string != ""} {
-        puts $channel "ROUTE VCC $vcc_route_string"
-    }
-    
     if {[llength $gnd_sinks] > 0} {
         puts $channel "INTERSITE GND [join $gnd_sinks]"
     }
     
-    if {$gnd_route_string != ""} {
-        puts $channel "ROUTE GND $gnd_route_string"
+    if {[llength $vcc_wires] > 0} {
+        puts "Printing VCC"
+        puts $channel "VCC $vcc_wires"
+        puts $channel "START_WIRES [get_static_source_wires $vcc_route_string $vcc_wires]"
+    }
+    
+    if {[llength $gnd_wires] > 0} {
+        puts "Printing GND"
+        puts $channel "GND $gnd_wires"
+        puts $channel "START_WIRES [get_static_source_wires $gnd_route_string $gnd_wires]"
+    }
+    
+    # re-enable the TCL display limit
+    tincr::reset_tcl_display_limit 
+}
+
+## Returns the source wires of VCC and GND nets (i.e. the wires that are connected to tieoff bels).
+#   This is done by parsing the ROUTE string of these nets, and grabbing the first wire within each
+#   independent section (which are separated by parenthesis "()")
+#
+# @param route_string The ROUTE string of the static net
+# @param net_wires The wires within the static net
+# @return A list of all source wires in the net
+proc get_static_source_wires {route_string net_wires} {
```

Comment:
Done. I also refactored it to take as input a net instead of the two parameters. This makes it more usable.